portal/views.py

```diff
@@ -236,3 +243,13 @@ def collection(request, slug, app_id=0):
     }
 
     return render(request, template, context=context)
+
+
+def events(request):
+    future_events = EventsObject.objects.filter(start_time__date__gt=timezone.now().date()).order_by("-start_time")
+    for event in future_events:
+        event.description_html = markdown.markdown(event.description)
+    past_events = EventsObject.objects.filter(start_time__date__lte=timezone.now().date()).order_by("-start_time")
```

Comment:
This means that upcoming events on the same day are marked as past events.

Reply:
Fixed.